tests/run_pretrained_models.py

```diff
@@ -190,6 +202,8 @@ def __init__(self, url, local, input_func, input_names, output_names,
         self.structured_outputs = structured_outputs  # Needed to determine output order for tf_function
         self.rtol = rtol
         self.atol = atol
+        self.ptol = ptol
```

Comment:
The new models in this commit contain post-processing with an ArgMax op so it makes more sense to allow p percent of the output tensor entries to differ from the TF model rather than allowing for a large atol or rtol.  The converted model has a small number of differing entries but they are not necessarily close to to their corresponding tf entries.